membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -182,7 +185,7 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
           RecurringContributionPaymentPlan = conSub.map(_.plan.name),
           MembershipJoinDate = memSub.map(_.startDate)
         )
-        res <- EitherT(tp.attrService.update(attributes).map(\/.right))
+        res <- EitherT(tp.attrService.update(attributes).map(_.disjunction)).leftMap(e => s"Dynamo failed to update the user attributes: $e")
```

Comment:
If you get a left as a DynamoReadError, I think you want `DynamoReadError.describe(e)` ?